e2e/specs/st_multiselect.spec.js

```diff
@@ -230,35 +197,24 @@ describe("st.multiselect", () => {
 
     it("calls callback if one is registered", () => {
       cy.get(".stMultiSelect")
-        .should("have.length", 10)
-        .last()
+        .eq(10)
```

Comment:
cy.get().eq() should be replaced by cy.getIndexed() to avoid flakiness

Reply:
Ohh was not aware that was a thing. Thanks! Will change that.